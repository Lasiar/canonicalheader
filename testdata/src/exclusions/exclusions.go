package exclusions

import "net/http"

func _() {
	h := http.Header{}
	h.Add("Exclusion", "") // want `use "exclusioN" instead of "Exclusion"`
	h.Add("exclusioN", "")
}
