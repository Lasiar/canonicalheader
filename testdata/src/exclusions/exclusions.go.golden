package exclusions

import "net/http"

func _() {
	h := http.Header{}
	h.Add("exclusioN", "") // want `use "exclusioN" instead of "Exclusion"`
	h.Add("exclusioN", "")
}
